pkgs/firehose/lib/src/github.dart

```diff
@@ -62,9 +67,14 @@ class Github {
             },
             body: body)
         .then((response) {
-      return response.statusCode != 201
-          ? throw RpcException(response.reasonPhrase!)
-          : response.body;
+      if (response.statusCode != 201) {
+        if (verbose) {
+          stderr.writeln('${response.statusCode} ${response.reasonPhrase}');
+          stderr.writeln(response.body);
```

Comment:
can we also print any headers? It looks like we should get some good ones that describe the issue

Reply:
done